rclcpp_tutorials/src/parameters/list_parameters.cpp

```diff
@@ -29,29 +29,29 @@ int main(int argc, char ** argv)
   auto node = rclcpp::Node::make_shared("list_parameters");
 
   // TODO(esteve): Make the parameter service automatically start with the node.
-  auto parameter_service = std::make_shared<rclcpp::ParameterService>(node);
+  //auto parameter_service = std::make_shared<rclcpp::ParameterService>(node);
```

Comment:
Instead of commenting out the parameter service I think it should be removed (including the comment above).

Same below.